src/integTest/kotlin/io/snyk/plugin/cli/ConsoleCommandRunnerTest.kt

```diff
@@ -84,7 +84,7 @@ class ConsoleCommandRunnerTest : LightPlatformTestCase() {
                         snykCliDownloaderService.isCliDownloading()
                     )
                     // No exception should happened while CLI is downloading and any CLI command is invoked
-                    val commands = ossService.buildCliCommandsList(listOf("test"))
+                    val commands = ossService.buildCliCommandsList_TEST_ONLY(listOf("test"))
```

Comment:
Why is this necessary? The naming does not adhere to java naming standards. Just marking it with `@TestOnly` should be enough.

Reply:
Unfortunately it's already cause call of buildCliCommandsList() in previous implementation of ContainerService (and it was hard to found that bug). 
To avoid such calls imho would be nice to have test_only restriction in the method name.